requirements.txt

```diff
@@ -1,5 +1,6 @@
 ecdsa==0.13
 enum34==1.1.3
+future>=0.15.2
 Jinja2==2.8
 lxml==3.4.4
```

Comment:
It appears lxml 3.6.1 supports python3.5..so should we be using this latest version?